WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /movierec/forms
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api-public.guidebox.com
DEBUG:requests.packages.urllib3.connectionpool:https://api-public.guidebox.com:443 "GET /v2/movies?sources=netflix&api_key=e9eb585ff0a9c36c22b6cf0fdc0a08cccfa5eac5&limit=10&offset=0 HTTP/1.1" 200 10550
DEBUG:root:Added 10 new movies to the database from NetFlix
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api-public.guidebox.com
DEBUG:requests.packages.urllib3.connectionpool:https://api-public.guidebox.com:443 "GET /v2/movies?sources=amazon_prime&api_key=e9eb585ff0a9c36c22b6cf0fdc0a08cccfa5eac5&limit=10&offset=0 HTTP/1.1" 200 9923
DEBUG:root:Added 10 new movies to the database from Amazon Prime
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api-public.guidebox.com
DEBUG:requests.packages.urllib3.connectionpool:https://api-public.guidebox.com:443 "GET /v2/movies?sources=hulu_plus&api_key=e9eb585ff0a9c36c22b6cf0fdc0a08cccfa5eac5&limit=10&offset=0 HTTP/1.1" 200 10065
DEBUG:root:Added 5 new movies to the database from Hulu Plus
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/genre/list?api_key=f2eee9cde7536b5ef17767e4e9a97239 HTTP/1.1" 200 547
DEBUG:root:Genres added to database: 19
INFO:root:There are currently 25 total movies in the database
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/271110?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 11098
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/7446?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 6877
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/424?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4210
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/269149?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5341
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/223702?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5157
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/278927?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5020
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/294272?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5114
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/127380?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4688
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/4951?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4800
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/318846?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 10380
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/365942?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8085
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/157336?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8286
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/334533?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4960
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/150689?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8240
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/258489?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 3725
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/333371?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4092
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/295699?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5075
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/264644?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 3738
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/339533?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 1809
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/300671?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 6034
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/9323?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 3635
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/14?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 7807
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/9377?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 1691
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/87101?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8465
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/273481?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 6376
DEBUG:root:Details for 25 movies retrieved successfully
WARNING:django.request:Not Found: /movierec/forms
WARNING:django.request:Not Found: /account
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /movierec/
WARNING:django.request:Not Found: /movierec/testing/
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api-public.guidebox.com
DEBUG:requests.packages.urllib3.connectionpool:https://api-public.guidebox.com:443 "GET /v2/movies?sources=netflix&api_key=e9eb585ff0a9c36c22b6cf0fdc0a08cccfa5eac5&limit=10&offset=0 HTTP/1.1" 200 10550
DEBUG:root:Added 10 new movies to the database from NetFlix
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api-public.guidebox.com
DEBUG:requests.packages.urllib3.connectionpool:https://api-public.guidebox.com:443 "GET /v2/movies?sources=amazon_prime&api_key=e9eb585ff0a9c36c22b6cf0fdc0a08cccfa5eac5&limit=10&offset=0 HTTP/1.1" 200 9923
DEBUG:root:Added 10 new movies to the database from Amazon Prime
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api-public.guidebox.com
DEBUG:requests.packages.urllib3.connectionpool:https://api-public.guidebox.com:443 "GET /v2/movies?sources=hulu_plus&api_key=e9eb585ff0a9c36c22b6cf0fdc0a08cccfa5eac5&limit=10&offset=0 HTTP/1.1" 200 10065
DEBUG:root:Added 5 new movies to the database from Hulu Plus
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/genre/list?api_key=f2eee9cde7536b5ef17767e4e9a97239 HTTP/1.1" 200 547
DEBUG:root:Genres added to database: 19
INFO:root:There are currently 25 total movies in the database
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/271110?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 11098
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/7446?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 6877
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/424?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4210
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/269149?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5341
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/223702?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5155
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/278927?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5020
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/294272?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5114
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/127380?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4687
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/4951?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4800
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/318846?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 10380
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/365942?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8085
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/157336?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8287
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/334533?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4960
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/150689?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8240
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/258489?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 3725
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/333371?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4092
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/295699?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5075
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/264644?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 3738
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/339533?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 1809
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/300671?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 6034
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/9323?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 3635
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/14?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 7807
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/9377?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 1691
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/87101?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8465
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/273481?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 6376
DEBUG:root:Details for 25 movies retrieved successfully
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /profile/
WARNING:django.request:Not Found: /account
ERROR:django.request:Internal Server Error: /account/signup/
Traceback (most recent call last):
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/template/response.py", line 107, in render
    self.content = self.rendered_content
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/template/response.py", line 82, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/template/response.py", line 64, in resolve_template
    return select_template(template, using=self.using)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/template/loader.py", line 53, in select_template
    raise TemplateDoesNotExist(', '.join(template_name_list), chain=chain)
TemplateDoesNotExist: account/signup.html
ERROR:django.request:Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 393, in login
    return LoginView.as_view(**defaults)(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/debug.py", line 76, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/auth/views.py", line 90, in dispatch
    return super(LoginView, self).dispatch(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/edit.py", line 174, in get
    return self.render_to_response(self.get_context_data())
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/auth/views.py", line 122, in get_context_data
    current_site = get_current_site(self.request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/shortcuts.py", line 15, in get_current_site
    return Site.objects.get_current(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/models.py", line 65, in get_current
    return self._get_site_by_request(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/models.py", line 50, in _get_site_by_request
    SITE_CACHE[domain] = self.get(domain__iexact=domain)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 379, in get
    self.model._meta.object_name
DoesNotExist: Site matching query does not exist.
ERROR:django.request:Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 393, in login
    return LoginView.as_view(**defaults)(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/debug.py", line 76, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/auth/views.py", line 90, in dispatch
    return super(LoginView, self).dispatch(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/edit.py", line 174, in get
    return self.render_to_response(self.get_context_data())
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/auth/views.py", line 122, in get_context_data
    current_site = get_current_site(self.request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/shortcuts.py", line 15, in get_current_site
    return Site.objects.get_current(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/models.py", line 65, in get_current
    return self._get_site_by_request(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/models.py", line 50, in _get_site_by_request
    SITE_CACHE[domain] = self.get(domain__iexact=domain)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 379, in get
    self.model._meta.object_name
DoesNotExist: Site matching query does not exist.
ERROR:django.request:Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 393, in login
    return LoginView.as_view(**defaults)(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/debug.py", line 76, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/auth/views.py", line 90, in dispatch
    return super(LoginView, self).dispatch(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/edit.py", line 174, in get
    return self.render_to_response(self.get_context_data())
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/auth/views.py", line 122, in get_context_data
    current_site = get_current_site(self.request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/shortcuts.py", line 15, in get_current_site
    return Site.objects.get_current(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/models.py", line 65, in get_current
    return self._get_site_by_request(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/models.py", line 50, in _get_site_by_request
    SITE_CACHE[domain] = self.get(domain__iexact=domain)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 379, in get
    self.model._meta.object_name
DoesNotExist: Site matching query does not exist.
ERROR:django.request:Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 393, in login
    return LoginView.as_view(**defaults)(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/debug.py", line 76, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/auth/views.py", line 90, in dispatch
    return super(LoginView, self).dispatch(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/edit.py", line 174, in get
    return self.render_to_response(self.get_context_data())
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/auth/views.py", line 122, in get_context_data
    current_site = get_current_site(self.request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/shortcuts.py", line 15, in get_current_site
    return Site.objects.get_current(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/models.py", line 65, in get_current
    return self._get_site_by_request(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/models.py", line 50, in _get_site_by_request
    SITE_CACHE[domain] = self.get(domain__iexact=domain)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 379, in get
    self.model._meta.object_name
DoesNotExist: Site matching query does not exist.
ERROR:django.request:Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 393, in login
    return LoginView.as_view(**defaults)(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/debug.py", line 76, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/auth/views.py", line 90, in dispatch
    return super(LoginView, self).dispatch(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/views/generic/edit.py", line 174, in get
    return self.render_to_response(self.get_context_data())
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/auth/views.py", line 122, in get_context_data
    current_site = get_current_site(self.request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/shortcuts.py", line 15, in get_current_site
    return Site.objects.get_current(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/models.py", line 65, in get_current
    return self._get_site_by_request(request)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/contrib/sites/models.py", line 50, in _get_site_by_request
    SITE_CACHE[domain] = self.get(domain__iexact=domain)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/quinn/461L/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 379, in get
    self.model._meta.object_name
DoesNotExist: Site matching query does not exist.
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api-public.guidebox.com
DEBUG:requests.packages.urllib3.connectionpool:https://api-public.guidebox.com:443 "GET /v2/movies?sources=netflix&api_key=e9eb585ff0a9c36c22b6cf0fdc0a08cccfa5eac5&limit=10&offset=0 HTTP/1.1" 200 10550
DEBUG:root:Added 10 new movies to the database from NetFlix
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api-public.guidebox.com
DEBUG:requests.packages.urllib3.connectionpool:https://api-public.guidebox.com:443 "GET /v2/movies?sources=amazon_prime&api_key=e9eb585ff0a9c36c22b6cf0fdc0a08cccfa5eac5&limit=10&offset=0 HTTP/1.1" 200 9831
DEBUG:root:Added 10 new movies to the database from Amazon Prime
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api-public.guidebox.com
DEBUG:requests.packages.urllib3.connectionpool:https://api-public.guidebox.com:443 "GET /v2/movies?sources=hulu_plus&api_key=e9eb585ff0a9c36c22b6cf0fdc0a08cccfa5eac5&limit=10&offset=0 HTTP/1.1" 200 10065
DEBUG:root:Added 6 new movies to the database from Hulu Plus
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/genre/list?api_key=f2eee9cde7536b5ef17767e4e9a97239 HTTP/1.1" 200 547
DEBUG:root:Genres added to database: 19
INFO:root:There are currently 26 total movies in the database
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/271110?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 11098
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/424?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4209
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/269149?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5340
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/223702?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5154
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/278927?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5024
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/7446?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 6876
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/294272?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5115
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/127380?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4690
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/4951?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4794
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/318846?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 10377
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/615?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5356
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/365942?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8085
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/334533?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4960
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/150689?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8242
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/258489?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 3722
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/264644?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 3738
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/9323?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 3640
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/157336?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8287
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/333371?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 4086
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/295699?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 5075
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/339533?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 1808
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/300671?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 6036
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/14?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 7810
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/9377?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 1689
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/87101?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 8467
DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): api.themoviedb.org
DEBUG:requests.packages.urllib3.connectionpool:https://api.themoviedb.org:443 "GET /3/movie/273481?api_key=f2eee9cde7536b5ef17767e4e9a97239&append_to_response=credits%2Cvideos%2Ckeywords HTTP/1.1" 200 6376
DEBUG:root:Details for 26 movies retrieved successfully
WARNING:django.request:Not Found: /movierec/form/images/hulu.png
